Makefile

```diff
@@ -153,12 +166,14 @@ build-logger-linux:
 .PHONY: build-%
 build-%:
 	$(KANIKO_DOCKER_CMD) kaniko \
+		--build-arg VERSION="$(VERSION)" \
+		--build-arg COMMIT="$(GIT_VERSION)" \
 		--dockerfile /workspaces/brigade/v2/$*/Dockerfile \
 		--context dir:///workspaces/brigade/ \
 		--no-push
 
 .PHONY: push-images
-push-images: push-apiserver push-scheduler push-observer push-logger-linux
+push-images: push-apiserver push-scheduler push-observer push-logger-linux push-git-initializer
```

Comment:
Push of the worker image is missing here. It's not a problem introduced by this PR, but would you mind fixing it right here?